### PR TITLE
Feature/add sort by feature

### LINKED ISSUE #104: [FEATURE]
I'd like to add a sort by option in campground page according to the price and rating.
@Vignesh025  Can I work on it?

### PR DESCRIPTION
**Pull request by Praju2002**
this fixes #104

### CODE REVIEW

**Comment by Sowmiyajayakumar** [comment]
Hey @Praju2002, it would be better if we implement sort by feature with following 4 options instead of restricting it to only 2:

1. Price - Low to High
2. Price - High to Low 
3. Reviews - Highest rated first
4. Reviews - Lowest rated first

**Comment by Praju2002** [comment]
Okay

File: views/campgrounds/index.ejs [hunk 1]
```diff
@@ -6,15 +6,20 @@
 
 <div class="d-flex justify-content-between align-items-center mb-3">
     <!-- Search Bar (left-aligned) -->
-    <form action="/campgrounds/search" method="GET" class="form-inline my-2 my-lg-0" id="search-form">
-        <div class="input-group">
-            <input class="form-control" type="search" placeholder="Search for campsites..." aria-label="Search" name="q" id="search-input">
-            <div class="input-group-append">
-                <button class="btn btn-info" type="submit" id="search-button">Search</button>
-            </div>
-        </div>
```
Comment (Vignesh025) [inline views/campgrounds/index.ejs:15]: Undo these deletions.

File: views/campgrounds/index.ejs [hunk 2]
```diff
@@ -6,15 +6,20 @@
...
+        <select name="sort" id="sort" class="btn btn-secondary">
+            <option value="">--Select--</option>
+            <option value="priceAsc" <%= (typeof query.sort !== 'undefined' && query.sort === 'priceAsc') ? 'selected' : '' %>>Price: Low to High</option>
+            <option value="priceDesc" <%= (typeof query.sort !== 'undefined' && query.sort === 'priceDesc') ? 'selected' : '' %>>Price: High to Low</option>
+            <option value="reviewsHighest" <%= (typeof query.sort !== 'undefined' && query.sort === 'reviewsHighest') ? 'selected' : '' %>>Reviews: Highest Rated</option>
+            <option value="reviewsLowest" <%= (typeof query.sort !== 'undefined' && query.sort === 'reviewsLowest') ? 'selected' : '' %>>Reviews: Lowest Rated</option>
+        </select>
+        <button type="submit" class="btn btn-primary">Sort</button>
     </form>
-    <button id="show-nearest-btn" class="btn btn-info">Show Nearest to Me</button>
```
Comment (Vignesh025) [inline views/campgrounds/index.ejs:17]: Undo this deletion too.

**Comment by Praju2002** [comment]
okay

**Comment by Vignesh025** [comment]
Make the PR to the ``test`` branch and not the ``main`` branch.

**Comment by Praju2002** [comment]
ok

**Pull request merged**